modules/request-quota-increase/generate_code.rb

```diff
@@ -0,0 +1,68 @@
+#################################################################
+# REQUEST QUOTA INCREASE CODE GENERATOR
+# This script generates the code for add more resources into the
+# module. It only generates the code for the resources that
+# 'Adjustable' is true.
+# Usage:
+# ruby generate_code.rb
+#################################################################
+
+require 'json'
+
+SERVICE = 'iam'
+
+# Some services are not available in all regions. For example, 'route53' and 'iam'
+# are only available in 'us-east-1'
+REGION = 'us-east-1'
```

Comment:
Perhaps allow these to be passed in as CLI args?

```
ruby generate_code.rb <SERVICE> <REGION>
```

Reply:
Good idea!